plugins/com.google.cloud.tools.eclipse.appengine.deploy/src/com/google/cloud/tools/eclipse/appengine/deploy/AppEngineDeployInfo.java

```diff
@@ -12,17 +18,14 @@
 import org.w3c.dom.NodeList;
 import org.xml.sax.SAXException;
 
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.IOException;
-import java.io.InputStream;
-
-import javax.xml.parsers.DocumentBuilderFactory;
-import javax.xml.parsers.ParserConfigurationException;
+import com.google.cloud.tools.eclipse.util.status.StatusUtil;
 
 /**
  * Parses and returns project ID and version from appengine-web.xml
+ *
+ * @deprecated Deployment should not rely on the project Id and version set in the appengine-web.xml
  */
+@Deprecated
```

Comment:
Hmm, I think you want `@Deprecated` on those specific elements — this will cause warnings for any reference to `AppEngineDeployInfo`.